src/instructions/RabbitizerInstruction/RabbitizerInstruction_Disassemble.c

```diff
@@ -126,6 +126,10 @@ bool RabbitizerInstruction_mustDisasmAsData(const RabbitizerInstruction *self) {
                     return true;
                 }
                 break;
+            case RABBITIZER_INSTR_ID_r5900_vclipw:
+                return true;
+            case RABBITIZER_INSTR_ID_r5900_vsqrt:
+                return true;
```

Comment:
Oh right I forgot about this function :sweat_smile: 
It is becoming the r5900 monkey patch lol

Could you add a brief comment explaining why those two instruction should be decomped as .word?